sdk/audit/metadata_adding_interceptor.go

```diff
@@ -46,3 +47,33 @@ func MetadataAddingClientInterceptor(
 
 	return err
 }
+
+func MetadataAddingConnectInterceptor() connect.UnaryInterceptorFunc {
```

Comment:
When/why is this being used?

Reply:
not sure, i think it just adds request ids to the metadata for tracking. both this and the credentials interceptors already existed for grpc, this is just making an equivalent one for connectrpc